src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) {
+        try {
+            Files.write(Paths.get(filePath), data.getBytes(StandardCharsets.UTF_8));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file: " + filePath, e);
+        }
+    }

```

Comment:
Consider using try-with-resources to ensure that all resources are closed properly. Although Files.write() is a static utility method that closes resources internally, it's a good practice to use try-with-resources when dealing with file operations to automatically handle resource management.